src/bitmessagemain.py

```diff
@@ -16,10 +16,12 @@
 os.chdir(app_dir)
 sys.path.insert(0, app_dir)
 
+
 import depends
 depends.check_dependencies()
```

Comment:
Please write imports in a order.
standard imports first.
from statements next.

custom from statements in the end.

Reply:
This is a special situation as `depends` checks which modules are available and handles missing modules gracefully. It needs to run before other imports.

Reply:
Also, since this is the main program function, there are no problems with potential recursive imports and things like that.